scripts/install_and_test_openqaoa_azure.sh

```diff
@@ -0,0 +1,23 @@
+# The script will stop if any intermediate file raises an error.
+set -e
+
+# TODO: Dynamically generate names (The order of installing and pushing is important here.)
+modulesList=("openqaoa-core" "openqaoa-qiskit" "openqaoa-azure")
+
+for entry in "${modulesList[@]}"; do
+    echo "processing src/$entry/setup.py"
+    cd src/$entry
+    pip install .
+    cd "../.."
+done
+
+for entry in "${modulesList[@]}"; do
+    echo "testing $entry"
+    cd src/$entry
+    pytest -n auto tests
```

Comment:
I was testing out using the parallel pytest execution code for this PR. Basically we just run `-n auto` so that the tests will be executed in parallel. Its in the docs of `pytest-xdist`

Reply:
Did you test it locally? Does it seem to work?

Reply:
Locally it worked. I had to change a couple of unittests. I updated `test_dev.yml` in this PR. with this newer line and everything seemed to work as well.